data.kidsfirstdrc.org/manifest.json

```diff
@@ -6,16 +6,15 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "quay.io/cdis/arborist:2.3.2",
-    "fence": "quay.io/cdis/fence:4.8.0",
-    "indexd": "quay.io/cdis/indexd:2.5.0",
-    "peregrine": "quay.io/cdis/peregrine:2.1.1",
-    "pidgin": "quay.io/cdis/pidgin:1.2.0",
-    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
-    "sheepdog": "quay.io/cdis/sheepdog:2.1.2",
-    "portal": "quay.io/cdis/data-portal:2.22.10",
-    "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "jupyterhub": "quay.io/occ_data/jupyterhub:master"
+    "arborist": "quay.io/cdis/arborist:2020.05",
```

Comment:
Hi @diw2 ,
What about `2020.07`?

Reply:
We are doing indexd migration and kidsfirst team is testing everything in staging before prod. That's why I kept the same version as staging. Maybe we could release to 2020.05 for now and communicate with kf for stable release after this is done? What do you think?   @paulineribeyre

Reply:
we are releasing 2020.05 because that's what has been going through the QA/staging/prod release process. We're not going to release 2020.05 in QA and 2020.07 in prod lol

Reply:
So just QA 2020.07 in lower tiers and deploy 2020.07 to prod (?) lol rofl